src/nopoll_conn.c

```diff
@@ -1346,6 +1346,22 @@ int           nopoll_conn_get_id (noPollConn * conn)
 	return conn->id;
 }
 
+/**
+ * @brief Allows to get the get_url from the connection
+ *
+ * @param conn The connection from where the url will be obtained
+ *
+ * @return The get_url or /
+ */
+const char * nopoll_conn_get_requested_url (noPollConn * conn)
+{
+        if (conn->get_url == NULL)
+            return "/";
+        else
```

Comment:
Hello, thanks for reporting,

Can you update the PR to make it:

if (! conn)
      return NULL;
return conn->get_url == NULL ? "/" : conn->get_url;

Thank you,
Best,